returnn/datasets/meta.py

```diff
@@ -1406,13 +1407,17 @@ def __init__(
           Now it could happen that ceildiv(data_len1 + data_len2, 6) < align_len1 + align_len2.
           This option would repeat intermediate ending frames such that data_len1 % 6 == 0,
           by setting it to {"data": 6}.
+        :param dict[str,(str,int)]|None pad_truncate_data_to_multiple_of: data_key -> (target_key, multiple).
+            to make sure the alignment is still valid after concatenation, we force length of every seq in data_key
+            to be length of seq in target_key * multiple
```

Comment:
I find the documentation a bit misleading. Specifically "to make sure the alignment is still valid after concatenation". Specifically, the option `repeat_in_between_last_frame_up_to_multiple_of` is also there to handle this issue already. So you should better explain, why is this new option needed. What would be a situation where `repeat_in_between_last_frame_up_to_multiple_of` would not work.